tests/e2e/e2e_helm_install_test.go

```diff
@@ -15,30 +15,32 @@ var _ = OSMDescribe("Test osm control plane installation with Helm",
 	func() {
 		Context("Helm install using default values", func() {
 			It("installs osm control plane successfully", func() {
-				if Td.InstType == NoInstall {
-					Skip("Test is not going through InstallOSM, hence cannot be automatically skipped with NoInstall (#1908)")
-				}
-
-				namespace := "helm-install-namespace"
-				release := "helm-install-osm"
-
-				// Install OSM with Helm
-				Expect(Td.HelmInstallOSM(release, namespace)).To(Succeed())
-
-				configmap, err := Td.GetConfigMap("osm-config", namespace)
-				Expect(err).ShouldNot(HaveOccurred())
-
-				// validate osm configmap
-				Expect(configmap.Data["permissive_traffic_policy_mode"]).Should(Equal("false"))
-				Expect(configmap.Data["egress"]).Should(Equal("false"))
-				Expect(configmap.Data["envoy_log_level"]).Should(Equal("error"))
-				Expect(configmap.Data["enable_debug_server"]).Should(Equal("false"))
-				Expect(configmap.Data["prometheus_scraping"]).Should(Equal("true"))
-				Expect(configmap.Data["tracing_enable"]).Should(Equal("false"))
-				Expect(configmap.Data["use_https_ingress"]).Should(Equal("false"))
-				Expect(configmap.Data["service_cert_validity_duration"]).Should(Equal("24h"))
-
-				Expect(Td.DeleteHelmRelease(release, namespace)).To(Succeed())
+				// TODO: install using MeshConfig manifest
+
+				// if Td.InstType == NoInstall {
+				// 	Skip("Test is not going through InstallOSM, hence cannot be automatically skipped with NoInstall (#1908)")
+				// }
+
+				// namespace := "helm-install-namespace"
+				// release := "helm-install-osm"
+
+				// // Install OSM with Helm
+				// Expect(Td.HelmInstallOSM(release, namespace)).To(Succeed())
+
+				// configmap, err := Td.GetConfigMap("osm-config", namespace)
```

Comment:
I guess you are not yet done here :)

Reply:
Right. configmap won't be used any more. We will do helm install without the configmap template, and configure using CRD.

One more thing I want to try to test is, simulate a scenario that configmap still exist in k8s storage, and `osm install` auto converts it to CRD